text/0046-metadata-for-offline-signers.md

```diff
@@ -85,17 +97,40 @@ enum MetadataDescriptor {
 }
 
 struct MetadataDescriptorV1 {
-  extrinsic_metadata: ExtrinsicMetadata
-  spec_version: String
-  spec_name: String
+  call_ty: u32, // TypeId
+  signed_extensions: Vec<SignedExtensionsMetadata>,
+  spec_version: String,
+  spec_name: String,
   ss58_prefix: u16,
   decimals: u8,
   token_symbol: String
 }
+
+struct SignedExtensionMetadata {
+  identifier: String,
+  ty: u32, // TypeId
+  additional_signed: u32, // TypeId
+}
 ```
 
 constitute metadata descriptor. This is minimal information that is, together with (shortened) types registry, sufficient to decode any signable transaction.
 
+**Note on `spec_version`**: this type is described in metadata; it is indeed u32 in Polkadot and in Kusama and most other networks and thus could be defined in typesafe manner, but theoretically it could be anything, probably even enum similar to how Era is constructed - why not? - as long as it is serializable. Something like 
```

Comment:
```suggestion
**Note on `spec_version`**: this type is described in metadata; it is indeed u32 in Polkadot and Kusama and most other networks and thus could be defined in a typesafe manner, but theoretically, it could be anything, as long as it is serializable. Something like 
```

Reply:
Removing the example would work as well.